Update installation.md for Pest 0.2 release

alexmartinfr opened the pull request:
The installation guide is now split into two separate sections:

- Install Pest on a blank project

- Install Pest on an existing project

alexmartinfr committed:
Update installation.md for Pest 0.2 release

The installation guide is now split into two separate sections:

- Install Pest on a blank project

- Install Pest on an existing project

alexmartinfr commented:
We might want to change the first screenshot of the *Blank project guide* with one showing Pest after running `pest --init`.

Feel free to commit changes directly to this PR is you see something to improve!

nunomaduro commented on source/docs/installation.md:
```diff
@@ -7,36 +7,96 @@ section: content
 # Installation
 
-**Requires [PHP 7.3+](https://php.net/releases/)**
+## Install Pest on a blank project
```
This should be on a guide instead. The installation should be for 90% use cases. Therefore, only the `Install Pest on an existing project` should be here.

alexmartinfr commented:
Seeing Windows has issues with the caret character, what if we put the full composer version syntax on the command?

Example, turn that:
```composer require phpunit/phpunit:"^9.0" --dev --update-with-dependencies```

Into that:
```composer require phpunit/phpunit:">=9.0.0 <10.0.0" --dev --update-with-dependencies```

[See this issue for more info.](https://github.com/pestphp/pest/issues/75)

alexmartinfr committed:
Update installation.md

Extracted everything related to installing Pest on a blank project, this will be a proper guide.
